src/cip/connectionManager/NetworkConnectionParametersBuilder.h

```diff
@@ -0,0 +1,64 @@
+//
+// Created by Vincent Prince on 05/22/20.
+//
+
+#ifndef EIPSCANNER_CIP_CONNECTIONMANAGER_NETWORKCONNECTIONPARAMETERSBUILDER_H
+#define EIPSCANNER_CIP_CONNECTIONMANAGER_NETWORKCONNECTIONPARAMETERSBUILDER_H
+
+#include "cip/Types.h"
+
+namespace eipScanner {
+namespace cip {
+namespace connectionManager {
+
+	class NetworkConnectionParametersBuilder {
```

Comment:
This a great idea to make a builder for the connections parameters! One notice, our names of the methods starts from the low case: SetRedundantOwner -> setRedundantOwner

Reply:
oops, I'm mixing our coding conventions with yours, I'll correct it.